### PR TITLE
Windows 10 mobile support

### LINKED ISSUE #82: Getting "There is an error in XML document (2,2)".
After making winium setup ready. which looks like
screen shot 2015-09-03 at 2 12 38 pm

The driver is running on port 9999 and test automation server under test in installed i.e by placing "AutomationServer.Instance.InitializeAndStart();" this in AUT Code  and also added reference to Inner Server in AUT project

And on running the TestCase getting an error which looks like 
screen shot 2015-09-03 at 2 22 45 pm

As i am trying to run my First Test Case using Winium . Am i missing soemthing ? Please help me out.

### PR DESCRIPTION
**Pull request by NickAb**
Adds support of Windows 10 UWP apps deployment to Mobile Emulator

Fixes #82

### CODE REVIEW

**Comment by NickAb** [comment]
We could try out https://msdn.microsoft.com/en-us/library/system.reflection.emit.typebuilder.definemethodoverride.aspx instead of copying the MS code we needed to patch

**Comment by NickAb** [comment]
@skyline-gleb ping

File: Winium/Winium.Mobile.Connectivity/IDeployer.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
+namespace Winium.Mobile.Connectivity
+{
+    #region
+
+    using System.Collections.Generic;
+
+    #endregion
+
+    public interface IDeployer
```
Comment (skyline-gleb) [inline Winium/Winium.Mobile.Connectivity/IDeployer.cs]: блудный сын вернулся ))

File: Winium/Microsoft.Phone.Tools.Deploy.Patched/Utils.cs [hunk 2]
```diff
@@ -0,0 +1,541 @@
...
+        {
+            try
+            {
+                var entry = new ZipArchive(packageStream).GetEntry(AppXManifestFile);
+                if (entry == null)
+                {
+                    throw new Exception(Resources.ExceptionManifestNotFound);
+                }
+
+                // NOTE: Patched version
```
Comment (skyline-gleb) [inline Winium/Microsoft.Phone.Tools.Deploy.Patched/Utils.cs]: оп

**Comment by rahulLiving** [comment]
I tried running on WIndows 10 emulator. Got the below error 
screen shot 2016-04-20 at 7 48 50 pm

**Comment by NickAb** [comment]
Please see https://github.com/2gis/Winium.StoreApps/issues/116#issuecomment-162839134
It might help, if not, report in the #116 issue

**Pull request merged**